projects/plugins/jetpack/tests/e2e/lib/pages/wp-admin/recommendations.js

```diff
@@ -0,0 +1,180 @@
+/**
+ * Internal dependencies
+ */
+import WpPage from '../wp-page';
+
+export default class RecommendationsPage extends WpPage {
+	constructor( page ) {
+		const url = `${ siteUrl }/wp-admin/admin.php?page=jetpack#/recommendations`;
+		super( page, { expectedSelectors: [ '[class^=jp-recommendations-]' ], url } );
+	}
+
+	// selectors section
+
+	get siteTypeCheckboxesSel() {
+		return '.jp-recommendations-question__site-type-checkboxes';
+	}
+
+	get siteTypePersonalCheckboxSel() {
+		return '.jp-checkbox-answer__container:nth-child(1) input';
```

Comment:
This locator would break if the order of the elements would change. 

Ideally, we should avoid ambiguous locators and rely on well-defined ones such as id's class names or specific data attributes.

In calypso, folks adding `data-e2e-*` attributes to avoid locators like these.